app/views/screens/LoginScreen.js

```diff
@@ -1,19 +1,19 @@
-import React, { useState } from "react";
+import React, { useState, useContext } from "react";
 import { Image, StyleSheet, Text, View } from "react-native";
 import { Button, TextInput } from "react-native-paper";
 
 import colors from "../styles/colors";
 import defaultStyles from "../styles/styles";
 import Screen from "../components/Screen";
+import AuthContext from "../../auth/context";
 
-function LoginScreen({ setIsLoggedIn }) {
+function LoginScreen(props) {
+  const authContext = useContext(AuthContext);
   const [username, setUsername] = useState("");
   const [password, setPassword] = useState("");
 
   function authenticate() {
-    setIsLoggedIn(true);
-    // if(username === 'admin' && password === 'admin')
-    //   setIsLoggedIn(true);
+    authContext.setUser({});
```

Comment:
Curious to know what does this do? 
How about we setUser to undefined instead of empty object?

Reply:
As discussed, we would be using the login object instead of user object to identify if user has successfully logged in or not. This PR isn't focused on authentication, so this logic will be updated in the upcoming auth PR.